src/main/java/uk/gov/hmcts/reform/sendletter/api/Letter.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sendletter.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Map;
+
+public class Letter {
+
+    public final List<Document> documents;
+
+    public final String type;
+
+    @JsonProperty("additional_data")
+    public final Map<String, Object> additionalData;
+
+    public Letter(
+        @JsonProperty("documents") List<Document> documents,
+        @JsonProperty("type") String type,
+        @JsonProperty("additional_data") Map<String, Object> additionalData
```

Comment:
those are not needed in constructor. Assuming this model remains being used for requests only.

Reply:
the additionalData?

Reply:
all of them